cmd/aries-agent-mobile/pkg/wrappers/constants.go

```diff
@@ -12,4 +12,31 @@ const (
 
 	// HandlerActions is the name of the actions operation
 	HandlerActions = "Actions"
+
+	// HandlerSendProposal is the name of the send proposal operation
+	HandlerSendProposal = "SendProposal"
+
+	// HandlerSendProposalWithOOBRequest is the name of the send proposal with out-of-bound request operation
+	HandlerSendProposalWithOOBRequest = "SendProposalWithOOBRequest"
```

Comment:
I think it makes sense to expose constants from commang.go file and use them here.

Reply:
i was wondering where to put these. they're used by both command and rest packages which are independent of each other. is it okay to move them to `command/command.go` and use them in `rest/*.go` as `command.HandlerX`?

Reply:
I meant to expose them in `pkg/controller/command/introduce/command.go`. And then reuse them from Aries.

Reply:
@soluchok you mean a new file `pkg/controller/command/commands.go` ? Agreed

Reply:
maybe expose these: https://github.com/hyperledger/aries-framework-go/blob/master/pkg/controller/command/introduce/command.go#L58-L67

Reply:
and these: https://github.com/hyperledger/aries-framework-go/blob/master/pkg/controller/rest/introduce/operation.go#L28-L38

Reply:
@ToJen yes